app/views/layouts/application.html.erb

```diff
@@ -78,5 +78,12 @@
         Feel better. Get Better.</a>
       </div>
 
```

Comment:
This is where the magic happens and we dynamically include a script tag on the page __ONLY IF__ there is a validation error.

We're using jQuery to "click" the contribute link.  If you're not familiar with jQuery or JavaScript, we'll go into what this is doing in a lot more detail later in the course; this would be normal if it wasn't entirely clear what this is doing.